integration-cli/docker_utils_test.go

```diff
@@ -120,6 +120,11 @@ func newRequestClient(method, endpoint string, data io.Reader, ct, daemon string
 
 func deleteContainer(container ...string) error {
```

Comment:
@AkihiroSuda ah, you are right. At some point these will be elsewhere so they are only used for teardown but yeah, that make sense 😓